File: ws/maza/u2/WS15/ZooApp/src/main/java/ec/edu/espe/zooapp/model/Cage.java

package ec.edu.espe.zooapp.model;

import java.util.ArrayList;

/**
 *
 * @author Alexander Maza
 */
public class Cage {
    private int id;
    private ArrayList<Animal> animals;

    @Override
    public String toString() {
        return "Cage{" + "id=" + id + ", animals=" + animals + '}';
    }

    
    
    public Cage(int id, ArrayList<Animal> animals) {
        this.id = id;
        this.animals = animals;
    }

    
    /**
     * @return the id
     */
    public int getId() {
        return id;
    }

    /**
     * @param id the id to set
     */
    public void setId(int id) {
        this.id = id;
    }

    /**
     * @return the animals
     */
    public ArrayList<Animal> getAnimals() {
        return animals;
    }

    /**
     * @param animals the animals to set
     */
    public void setAnimals(ArrayList<Animal> animals) {
        this.animals = animals;
    }
    
}

File: hw/gongora/U2/hw19Exceptions/javascript/ec/edu/espe/zoo/model/Cage.mjs
export class Cage {
    #id;
    #animals;

    constructor(id, animals) {
        this.#id = id;
        this.#animals = animals;
    }

    toString() {
        return `Cage {id: ${this.#id}, animals: ${this.#animals.join(', ')}}`;
    }

    getId() {
        return this.#id;
    }

    setId(id) {
        this.#id = id;
    }

    getAnimals() {
        return this.#animals;
    }

    setAnimals(animals) {
        this.#animals = animals;
    }
}
